main.go

```diff
@@ -306,17 +304,20 @@ func getKeyStore(datastore string, keyName string) *Keystore {
 	return NewKeystore(dirstore, keyName)
 }
 
-func loadTenantToken(datastore string) ([]byte, error) {
+func loadTenantToken(config *menderConfig, datastore string) ([]byte, error) {
+
 	dirstore := NewDirStore(datastore)
-	raw, err := dirstore.ReadAll(defaultTenantTokenFile)
+
+	raw, err := dirstore.ReadAll(config.GetTenantToken())
```

Comment:
Just a thought, but this might be a good opportunity to clean up the config a bit, we have:

* `ArtifactVerifyKey` - seems to be an absolute path, 
* `DeviceKey` - relative path inside state dir, file name configurable (needed?)
* `UpdateLogPath` - absolute path to update logs?
* `TenantTokenPath` - relative inside state dir, file name configurable as well (really needed?)

Maybe we could remove device key and tenant token path from configuration and use fixed names instead. I don't think we've come up with a valid use cases for using different paths for these settings.

(Well, in theory I think there might be a case for using absolute paths for device key & tenant token, eg. when both files are accessible form some secure storage).